### PR TITLE
Module 1 labs - Enrichment and Filtering

### PR DESCRIPTION
**Pull request by manoellins**

### CODE REVIEW

File: rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java [hunk 1]
```diff
@@ -98,20 +99,32 @@ public static void main(String[] args) throws Exception {
 
     public static class EnrichmentFunction
             extends RichCoFlatMapFunction<TaxiRide, TaxiFare, RideAndFare> {
-
+        private ValueState<TaxiFare> fareState;
```
Comment (ness-PavelKhokhlov) [inline rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java]: should be `transient`
private transient
see details:
https://stackoverflow.com/a/72558397

File: rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java [hunk 2]
```diff
@@ -98,20 +99,32 @@ public static void main(String[] args) throws Exception {
...
+        private ValueState<TaxiRide> rideState;
```
Comment (ness-PavelKhokhlov) [inline rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java]: should be `transient`
private transient
see details:
https://stackoverflow.com/a/72558397

**Review by ness-PavelKhokhlov — COMMENTED** [review]
Hello @manoellins,
pls see my comments

File: rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java (hunk 1 above)
Comment (manoellins) [reply rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java]: done

File: rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java (hunk 2 above)
Comment (manoellins) [reply rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java]: done

**Pull request merged**